### PR TITLE
docs(api): Update example link

### PR DESCRIPTION
**Pull request by chenxsan**
Update example link which is broken.

### CODE REVIEW

File: src/content/api/node.mdx [hunk 1]
```diff
@@ -223,7 +224,7 @@ stats.toJson({
 All available options and presets are described in the stats [documentation](/configuration/stats).
 
 > Here’s an [example]
-(https://github.com/webpack/analyse/blob/master/app/pages/upload/example.json)
+(https://github.com/webpack/analyse/blob/master/app/pages/upload/example1.json)
```
Comment (EugeneHlushko) [inline src/content/api/node.mdx]:
```suggestion
(https://raw.githubusercontent.com/webpack/analyse/master/app/pages/upload/example1.json)
```

Comment (chenxsan) [reply src/content/api/node.mdx]: With link to raw content you can't go anywhere but the json file, maybe you wanna check around then? I can update it if you insist :)

Comment (montogeek) [reply src/content/api/node.mdx]: File is not displayed by GitHub, it is better to show it by default:
image

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**